Sources/SWBApplePlatform/Plugin.swift

```diff
@@ -40,7 +40,8 @@ struct TaskProducersExtension: TaskProducerExtension {
     }
 
     var unorderedPostSetupTaskProducers: [any TaskProducerFactory] {
-        [StubBinaryTaskProducerFactory()]
+        [StubBinaryTaskProducerFactory(),
+         AppIntentsMetadataTaskProducerFactory()]
```

Comment:
nit: formatting would be more readable as:

```swift
[
    StubBinaryTaskProducerFactory(),
    AppIntentsMetadataTaskProducerFactory()
]
```

especially as things are added over time.